docs/hub/spaces.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Documentation for Spaces
+---
+
+<h1>How to get started with Spaces</h1>
+
+> ⚠️ This feature is currently in private beta, reach out to [julien-c](https://huggingface.co/julien-c) if you'd like to try it out ⚠️
+
+## What are Spaces?
+
+Spaces are a simple way to host a ML demo app, directly on your user profile or your organization’s hf.co profile.
+
+We support two awesome SDKs that let you build cool apps in Python: **[Streamlit](https://streamlit.io/)** and **[Gradio](https://gradio.app/)**.
+
+**To get started**, simply click on [New Space](https://huggingface.co/new-space) in the top navigation menu, create a new repo of type `Space`, and pick your SDK:
+
+![/docs/assets/hub/new-space.gif](/docs/assets/hub/new-space.gif)
```

Comment:
Is it in `gif` format because you'd like an animation / screencast here ?

Reply:
yes it already is an animated gif